docs/source/overview/environments.rst

```diff
@@ -71,6 +75,9 @@ Classic environments that are based on IsaacGymEnvs implementation of MuJoCo-sty
 .. |cartpole-link| replace:: `Isaac-Cartpole-v0 <https://github.com/isaac-sim/IsaacLab/blob/main/source/extensions/omni.isaac.lab_tasks/omni/isaac/lab_tasks/manager_based/classic/cartpole/cartpole_env_cfg.py>`__
 .. |cartpole-rgb-link| replace:: `Isaac-Cartpole-RGB-Camera-v0 <https://github.com/isaac-sim/IsaacLab/blob/main/source/extensions/omni.isaac.lab_tasks/omni/isaac/lab_tasks/manager_based/classic/cartpole/cartpole_camera_env_cfg.py>`__
 .. |cartpole-depth-link| replace:: `Isaac-Cartpole-Depth-Camera-v0 <https://github.com/isaac-sim/IsaacLab/blob/main/source/extensions/omni.isaac.lab_tasks/omni/isaac/lab_tasks/manager_based/classic/cartpole/cartpole_camera_env_cfg.py>`__
```

Comment:
Hm is there a reason the env names here misse parity with direct environment counterparts?

That is, `Isaac-Cartpole-RGB-Camera-v0` vs. `Isaac-Cartpole-RGB-Direct-v0`

We should try to be consistent in the names

Reply:
Changed to be Isaac-Cartpole-RGB-v0 and the like